terms.html

```diff
@@ -185,4 +185,10 @@
 does not use the term URI or IRI because those terms have been deemed to be
 confusing to Web developers.
   </dd>
+  <dt><dfn data-lt="well-formed">well-formed</dfn></dt>
+  <dd>
+A well-formed document is a conforming document, 
+and an instance of the core data model defined via a context,
+which produces no JSON-LD processing errors.
```

Comment:
This definition is in conflict with the definition of well-formed relative to the credentialSchema property.

Here's the definition from wikipedia for xml well-formedness https://en.wikipedia.org/wiki/Well-formed_document :
> adheres to the [syntax](https://en.wikipedia.org/wiki/Syntax) rules specified by the [XML](https://en.wikipedia.org/wiki/XML) 1.0 specification in that it must satisfy both physical and logical structures

Perhaps more odd in your definition is the inclusion of JSON-LD processing, which is orthogonal to well-formedness. In the XML notion of well-formedness, the semantics are not included, just the syntax. 

> The rules of [validity](https://en.wikipedia.org/wiki/Validity_(logic)) mean that a document complies with the restraints stated within a DTD.

The semantics are a matter of validity, not syntax.

I suggest
```suggestion
A well-formed Verifiable Credential is a conforming document which satisfies the syntax defined by its credentialSchema property, if present.
```

Reply:
Hrm, this isn't just about VCs, it's about all document types specified by the spec (VCs and VPs).